CI/scripts/algorithm_names_map.dat

```diff
@@ -0,0 +1,52 @@
+# #############################################################################
+# Certifier Framework Encryption algorigthm names mapping
+# Specify the way hard-coded strings will be remapped based on some
+# organization.
+# #############################################################################
+#
+# enc_method:Encryption algorithms supported.
+#
+enc_method:"aes-128"
```

Comment:
This string will be replaced as: `enc_method_aes_128` and so on...

If we want better prefixes, it's easy to change this mapping file entry.